public/toolkit/icons/icomoon/demo.html

```diff
@@ -8,231 +8,194 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-<svg style="position: absolute; width: 0; height: 0; overflow: hidden;" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Could this file be added to gitignore?

Reply:
hm probably as long as it's still in there for people to copy over

Reply:
@akegan ah is this file used for anything? I was under the impression it doesn't do anything and is just for reference

If it's being used then it should probably be kept up to date.

Reply:
when we generate a new icomoon download we have to use demo.html to copy and paste values into _icons.html but I think you're right that we don't need it to be checked in